XamarinCommunityToolkit/Converters/NotEqualsConverter.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Checks whether the incoming value doesn't equal the provided parameter.
+    /// </summary>
+    public class NotEqualsConverter : IValueConverter
```

Comment:
nitpick: NotEqualConverter (Equal**S**), since we have EqualConverter

Reply:
@sthewissen just this is quite important, I guess (Make names consistent)

EqualConverted & NotEqualConverter

Alright?

Reply:
Hmm, thought I already fixed this one. Probably in my dreams then 🤔